packages/html-test-app/src/preview-examples/avatar-image.html

```diff
@@ -13,9 +13,9 @@
 
 <body class="theme-brand-dark">
   <!-- Preview code -->
-  <ix-avatar image="https://ui-avatars.com/api/?name=John+Doe"></ix-avatar>
+  <ix-avatar image="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADUAAAA1CAIAAABuhDQnAAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAANaADAAQAAAABAAAANQAAAADneQ/wAAAHVklEQVRoBe1YaWwUZRje2dndmb13u7stvUtbyiKl5SoIiECk1oYCUokkRUFENDGGGMAr/DAaFRJj1B+gEA0o2hBBU4ptBCGRgmApIEI5ihS6LT326t67s8fM+NVpp9NhZvYoP/jBdJJ9v/d73ud95v2O+aaQaM8e0UN8iR9ibUPSHukb3wA97PWTpPx4xW5Xzs0b/lu3vA6n024fdLlJkqTYIAhK0+sMJpPGaFCVlNwzT7mt06eWCEp2/UpE5PyrV7saG7u7exJPmZeXW7Bixdlp02IiKPEogExOX4Wly15f39VlSSoHDS4oyDfV1bXlF9CeuAYsWr48LggAIJJcfPrU2d1fud2eRPCcGBDbd/rMYq3akpcvghIqZEL6EIKYdbD+z6NNnFmTdXZdbZ/r9zqmluIJSExo/VYcPdJ66nSyOgTwrX+0AE4BAN0VX9+ii21nmn+jAx6UATgBc1y2OPoKPe4z3+6Ly5IaADADfuHYOPpMzU04jgtTpNwLmAG/cLjQ/lxqt4KJAuIlEnjezBnCRAn2+gPBv69dp8GAv3Tp0nZTBu1hGUL60i4Mzw+dRtNy+EdWZGrNm513piypZsYOZamuYXqYNu/4gg2v42wrE/pA7Mx0E4sHZAG5WE66yavPPOiw2mwUTiGX0wHjNLRqdU7mBCYJyAJyMT1Mm1dfmtNJ43Im8M4PGpO48Vz10ywwMxeri1efyu2ioQtnTQ953Zw35vMSSS7w19etlUmlNDkwJB7edybv+lA5hgcXhsVVpcW91/5hMjJtMSxOLzKrDOyJxcQw7ZLCgi2bNuzcvXfUya+Pt35hDKPiX6p+KjdDKDeBE7bOm0lV8aO33ny2aimtjwiHaZtl8NaPws02F2+rq6VjYATVTS5TTMiRKtWxUABzWF03LkcDPiAxEgqiKjWNFDZgGD646/O1m7f93HxMGMlbP3CkKs7J/PrtNyQwTFFIVZq8Z1brzeWIztB4onXrjv27jrapHq+Sm4bWo0QmE87E6kVksp92f7F5wzqWn9XkPV89Ye9/f+0qo1ZDB2TMWYTojaD518VrO748YHe671j6uu5Zl9fW+O52GPImUshoDHd6AjGcBKcn8Bcj8CAW9foxcOiXSoYflUICT/WSJ1UKxS27wzKphE7ENDjGtwyVLlMha+pWK/1jlhUoGxXZcbubpgC2VK1FGY8RxfHfL3S03ejpsbn8wbAclWYaNNMn5dQuLFOgY5YtRbLttY3F7dcXhIJN/vAVLEozUwZb3wo1+n22TgJBPizqH4sN2QfU+cXAN6t88oFDx6ivodnl5rDLIUOVNFaByNZWzgY37Ylr1ORnVzoH3jGq1vW6G33D65KKYo9vfa4+/f9RiMSICBZiUmOOAWVWHozI0436kqJcMDoL55a9/Hyl/dxJVZoBUYxKZEYlYhORMB7yiyGoFJXudQWZIez6ZY6dIkwoHsZ6jv+imThZnpE9ozBjWuYisH6tJxuIaEReYmYiU7bvz87WdyIQrlWjIIFYzPH9QhKEp/MGuJkKYIlUIkOYnmRtkhg+YoLsrFj2/rKl33PcHyZI0dDyS+ySIOMSB5KEYzGQEeQF2Vk52fVz4ERtz6ASgj4wqlaysDzNSCjkd9hg6dD+B4nFqHp0S+KJYLtf7XE0DtgDXKcstj4qFEAvRYkE9YHRGfh3aMQhMZxpnspOnkD7bgjjFAdC2eNLsx32BCGYWz2NYRpiGM6aUqrQ6pnORGxnJNrm8fEhefVFSHIwmVkv1+rlGh1fGgF/k92Jc40sFSJUocaoeD0Psc0fsQyO2aiQQVwX4Hg9zJs6UY5w+GniBivv4RlghPR96vStNyGiCHvNg7AeF/bNuXt0jhFjzL5DORs+eWWkl+P3djD0q230oH4/gnd8ATRMkg0k96Pn64f2yLjXhDSNWiG0+2zvuBPlH1zAL6QPdL9r98XkHC8uo0o2JZ3Dz1K8bP5jLA+zCZbFoQE703O/HUcf2Ne3+nGwq90f+eKcbETC4aeRJbmmNUtm0k2W4YvhG67c5P2uHEELJaAwzb7Qfkgxgh/9NShl6+Zky2BuBjCy771QCcPcLyGCJOsuX7/mD4zS8Vjs8wsnrCUUqdDrcmPshZKlRSvytN0ubDA4em4D55qa+aUfbqw2anknwLaOO9/1DnDmYjmT+P/uZxnalREf/U9wJtEgLnUjaf5QOE2jLC/K0qs56k3hMYLY1N7xQ6+VGS5gC+0vrLCtVk+7TrFdipPRyJguCCorn4qqtWOcXI3+cGTVpfZWt5erk9vHPXu4sSLRPndwgRvvkGuGPi1GLn1WTlxxQRz/uNNibjmflDiQIYn6UXqs0VhNv2cmKt1hVBVhAY0x3ZBXOCKV47cvHDk8YNvZ2Q2Kx9Edz5W0PorwEhatuucCJ6rpMe9KwjJNrcxC0SxEZpBJwfseaOrDsKu+wBGbs83tjbuJCIhMUR/FCApy3u0Ft0CCcXYlN//GmSyF8Ef6UigaI+RR/RjFSMH8D1K1s1CBjxnfAAAAAElFTkSuQmCC"></ix-avatar>
```

Comment:
This makes the example unreadable. Also react/angular/vue examples are not updated.

Reply:
@silviowolf Maybe this one here will be fine -> https://www.gravatar.com/avatar/00000000000000000000000000000000 but remember to apply this in html-test-app/angular-test-app/react-test-app and vue-test-app